alf/algorithms/generator.py

```diff
@@ -311,6 +318,7 @@ def __init__(self,
                  critic_optimizer=None,
                  inverse_mvp_optimizer=None,
                  optimizer=None,
+                 lambda_optimizer=None,
```

Comment:
I think lambda_optimizer is not documented below

Reply:
updated.